src/cascadia/TerminalSettingsEditor/MainPage.xaml

```diff
@@ -161,6 +161,10 @@
                 <muxc:NavigationViewItem.Icon>
                     <FontIcon Glyph="&#xEA86;" />
                 </muxc:NavigationViewItem.Icon>
+                <muxc:NavigationViewItem.InfoBadge>
+                    <muxc:InfoBadge Visibility="{x:Bind ExtensionsVM.DisplayBadge, Mode=OneWay}"
+                                    Value="1" />

```

Comment:
I feel like using a number here is wrong, because that would indicate there's exactly 1 new thing to see in that subpage. We don't have other types of info badges in WinUI, do we?

Reply:
Fair. Yeah, an InfoBadge by default only supports:
- Value (`int`) --> show a number
- IconSource --> show an icon (we use Segoe Fluent Icons already)
I looked through the Segoe Fluent Icons and couldn't find any that looked good tbh. 🫤

We can retemplate the control though. I'll do that and put in "NEW" like PowerToys does it.